lighthouse-core/test/runner.js

```diff
@@ -58,14 +48,14 @@ describe('Runner', () => {
     const flags = {
       auditWhitelist: null
     };
-    const config = {
+    const config = new Config({
       passes: [{
-        gatherers: [new HTTPSGatherer()]
+        gatherers: ['https']
```

Comment:
there's no longer a chance of of getting an expanded gatherer passed back in, so you can just delete this test